### PR TITLE
Add 5 faculty members at Nanjing University

### PR DESCRIPTION
**Pull request by NJUJYB**
Add 8 faculty members at Nanjing University

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by emeryberger** [comment]
Please resolve conflicts and I will merge.

**Comment by NJUJYB** [comment]
Dear emeryberger, I have updated the commit, please check it.

**Comment by NJUJYB** [comment]
Dear emeryberger, I have tacked all of the conflicts, please check it.

**Comment by NJUJYB** [comment]
> Please resolve conflicts and I will merge.
Dear emeryberger, I have tacked all of the conflicts, please check it. I hope it could be merged. Thank you.

**Comment by NJUJYB** [comment]
@emeryberger Looking forward to your favourable reply.

File: csrankings.csv [hunk 1]
```diff
@@ -16969,6 +16975,7 @@ Yadati Narahari,IISc Bangalore,http://www.csa.iisc.ac.in/~hari,NOSCHOLARPAGE
 Yadong Mu,Peking University,http://www.muyadong.com,Fqqx4HsAAAAJ
 Yael Amsterdamer,Bar-Ilan University,https://u.cs.biu.ac.il/~amstery,70iHzEUAAAAJ
 Yafei Dai,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=5999,VSvWlvkAAAAJ
+Yafeng Yin,Nanjing University,http://cs.nju.edu.cn/yafeng/,v5CGgS8AAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: This person's web page says they are an "assistant researcher", which does not sound eligible for inclusion.

File: csrankings.csv [hunk 2]
```diff
@@ -14310,6 +14313,7 @@ Sheng Li 0001,University of Georgia,http://www.cs.uga.edu/~shengli,DEncVcYAAAAJ
 Sheng Li 0002,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=6082,NOSCHOLARPAGE
 Sheng Li 0003,Harbin Institute of Technology,http://ir.hit.edu.cn/~ls,NOSCHOLARPAGE
 Sheng Wei 0001,Rutgers University,http://eceweb1.rutgers.edu/~sw891,9fBhPy0AAAAJ
+Sheng Zhang 0001,Nanjing University,http://cs.nju.edu.cn/sheng/,2fqdkfMAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: "Assistant researcher": this title does not indicate that this person is eligible ("professor" and similar yes, this, not obviously so).

**Review by emeryberger — CHANGES_REQUESTED** [review]
My spot check turned up a couple of people who appear to be ineligible for inclusion: please clarify ("research faculty" are not currently included).

File: csrankings.csv (hunk 2 above)
Comment (NJUJYB) [reply csrankings.csv]: Thanks, I will recheck it and update it again.

**Comment by NJUJYB** [comment]
@emeryberger Dear emeryberger, I have removed three assistant researchers. I am really sorry for my mistake. Please check it again.

**Comment by NJUJYB** [comment]
@emeryberger Thanks a lot.

**Pull request merged**